### DISCUSSION TITLE
Whats the name of the feature/plugin that shadows text when doing a search?

### DISCUSSION THREAD

**Discussion by heysokam** [LazyVim/LazyVim#3969] [Q&A] [answered]
LazyVim has a feature where all text is "shadowed" or "darkened" when searching with `/` or `?`.
I want to remove that entirely, as it is very distracting for me personally,
but I don't know what to search for, so I cannot find how to do so.

What's the name of the thing? Is it a plugin or what is creating this behavior exactly?

**Comment by dpetka2001** [comment]
This was the old behavior of `flash.nvim` I believe, but not any more after a certain plugin update. Do you have all the plugins updated? Do you maybe have something in your personal configuration that enables this feature?

**Reply by heysokam** [reply]
I definitely didn't add it manually, 100% not.
It might be part of some old configuration taken from some old docs or something, though, as I haven't updated my config, just the plugins.
Will take a look

**Reply by heysokam** [reply]
Just checked. There is nothing in my config referencing `flash`

**Reply by dpetka2001** [reply]
Are all your plugins updated? Open Lazy UI and press `U` for all plugins to update to latest version and restart Neovim to check for behavior.

**Reply by heysokam** [reply]
![image](https://github.com/LazyVim/LazyVim/assets/7308253/7d3128d3-e557-4660-85d9-ba0ee8453200)

I keep my plugins up to date everyday.
It is still showing up in my plugins list.

**Reply by dpetka2001** [reply]
That is not your plugins list. That is LazyVim's plugins list. Do you use `flash` or maybe `leap`?

**Reply by heysokam** [reply]
Is there a way to fully disable a plugin enabled by lazyvim?

**Reply by heysokam** [reply]
> That is not your plugins list. That is LazyVim's plugins list. Do you use `flash` or maybe `leap`?

Well, that's what I meant. That it is not in my personal config, but it is still active in my plugins because lazyvim is installing it.

**Reply by dpetka2001** [reply, marked as answer]
You can see how to disable plugins in the [docs](https://www.lazyvim.org/configuration/plugins#-disabling-plugins). Just `enabled = false` for the plugin you want to disable.

**Reply by heysokam** [reply]
> You can see how to disable plugins in the [docs](https://www.lazyvim.org/configuration/plugins#-disabling-plugins). Just `enabled = false` for the plugin you want to disable.

Tysm for the link!
